deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinState.java

```diff
@@ -178,17 +181,14 @@ public TwinState(TwinCollection tags, TwinCollection desiredProperty, TwinCollec
      */
     public JsonElement toJsonElement()
     {
-        /* SRS_TWIN_STATE_21_002: [The toJsonElement shall return a JsonElement with the information in this class in a JSON format.] */
```

Comment:
Removed all spec comments, per Tim's assertion that they are not necessary and add little to no value.